javascript/javascript1/week2/candy_helper.js

```diff
@@ -0,0 +1,49 @@
+let boughtCandyPrices = [];
+function addCandy(candyType, weight) {
+    let pricePerGram = 0;
+    switch (candyType) {
+        case 'Sweet':
+            pricePerGram = 0.5;
+            break;
+        case 'Chocolote':
+            pricePerGram = 0.7;
+            break;
+        case 'Toffee':
+            pricePerGram = 1.1;
+            break;
+        case 'Chewing-gum':
+            pricePerGram = 0.03;
+            break;
+        default:
+            console.log('Wrong type, select something else');
+    }
+    let price = pricePerGram * weight;
+    boughtCandyPrices.push(price);
+}
+
+let amountToSpend = Math.random() * 100;
```

Comment:
always use const until it doesn't let you

Reply:
fixed